androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/currentpatients/CurrentPatientItem.kt

```diff
@@ -0,0 +1,141 @@
+import com.EENX15_22_17.digital_journal.android.ui.theme.danger
+import com.EENX15_22_17.digital_journal.android.ui.theme.primaryColor
+
+class CurrentPatientItem {
+
+    @Composable
+    fun PatientCard(
+        patientId: String = "1",
+        name: String = "PatientName",
+        securityNumber: String = "00000000",
+        navigateSelectedPatient: (patientId: String) -> Unit = {},
+        showOverview: (patientId: String) -> Unit = {}
+    ) {
+        var showWarning by remember { mutableStateOf(false) }
+        var showComments by remember { mutableStateOf(false) }
+        Card(
+            modifier = Modifier
+                .padding(horizontal = 120.dp, vertical = 2.dp)
+                .fillMaxWidth()
+                .clickable { navigateSelectedPatient(patientId) },
+            backgroundColor = primaryColor
+        ) {
+            Row(horizontalArrangement = Arrangement.SpaceBetween) {
+                Text(
+                    text = name,
+                    modifier = Modifier.padding(16.dp),
+                    fontSize = 20.sp,
+                    fontWeight = FontWeight.Bold
+                )
+                Text(
+                    text = securityNumber,
+                    modifier = Modifier.padding(16.dp),
+                    fontSize = 20.sp
+                )
+                Row(horizontalArrangement = Arrangement.End) {
+                    IconButton(onClick = { showOverview(patientId) }) {
+                        Icon(
+                            imageVector = Icons.Rounded.Info,
+                            contentDescription = "Overview",
+                            tint = colorIcon
+                        )
+                    }
+
+                    IconButton(
+                        onClick = { showComments = true }) {
+                        Icon(
+                            imageVector = Icons.Rounded.Notifications,
+                            contentDescription = "Comments",
+                            tint = colorIcon
+                        )
+                    }
+                    IconButton(onClick = { showWarning = true }) {
+                        Icon(
+                            imageVector = Icons.Rounded.Warning,
+                            contentDescription = "Warnings",
+                            tint = danger
+                        )
+                    }
+                    if (showWarning) {
+                        Dialog(onDismissRequest = { showWarning = false }) {
+                            WarningContent(patientId = patientId)
+                        }
+                    } else if (showComments) {
+                        Dialog(onDismissRequest = { showComments = false }) {
+                            CommentsContent(patientId = patientId)
+                        }
+                    }
+                }
+            }
+        }
+
+    }
+
+    @Composable
+    fun WarningContent(patientId: String) {
```

Comment:
This view seems like a PoC, but later on we should maintain font sizes (and other theme-related stuff) in a single location

Reply:
Added todo for this.